go.mod

```diff
@@ -4,12 +4,10 @@ go 1.15
 
 require (
 	github.com/go-logr/logr v0.1.0
-    	k8s.io/api v0.17.14
+	k8s.io/api v0.17.14
 	k8s.io/apimachinery v0.17.14
 	k8s.io/client-go v0.17.14
-	k8s.io/component-base v0.17.14
-	k8s.io/klog v1.0.0
-	k8s.io/utils v0.0.0-20201110183641-67b214c5f920
+	k8s.io/utils v0.0.0-20201110183641-67b214c5f920 // indirect
```

Comment:
These changes came from `go mod tidy`

Reply:
It should be committed separately I think, together with `go.sum`. And there should be a CI check for that.

Edit: #10